ansible/roles/init/tasks/main.yml

```diff
@@ -15,7 +15,7 @@
 
 - name: Add ppa Repository
   become: yes
-  apt_repository: repo=ppa:ondrej/{{ php_ppa }}
```

Comment:
@InFog  I'm happy to do this, but is it relevant?  Is it likely that this repo is likely to want configuring by the user/platform?